assets/queries/k8s/containers_with_sys_admin_capabilities/query.rego

```diff
@@ -10,9 +11,8 @@ CxPolicy[result] {
 	types = {"initContainers", "containers"}
 	containers := specInfo.spec[types[x]]
 
-	add := object.get(containers[index].securityContext.capabilities, "add", "undefined")
-	add != "undefined"
-	add[_] == "SYS_ADMIN"
+	add := common_lib.valid_key(containers[index].securityContext.capabilities, "add")
+	add == true
```

Comment:
It is missing the main verification of this query - if some key of  `add` is equal to "SYS_ADMIN". 

I suggest: `containers[index].securityContext.capabilities.add[_] == "SYS_ADMIN"`. It is unnecessary to check if the key is valid since the comparison itself already assumes that the key is defined and not null